tests/console/libqca2.pm

```diff
@@ -60,8 +60,10 @@ sub run {
     assert_script_run "$qca_cmd show cert cert.pem";
     assert_script_run "$qca_cmd keybundle make rsapriv.pem cert.pem --pass=suse --newpass=suse";
     assert_script_run "$qca_cmd keystore list-stores";
-    script_run "$qca_cmd keystore monitor";
+    type_string "$qca_cmd keystore monitor\n";
+    sleep(1);
```

Comment:
That looks like it should wait for the prompt to appear instead . Separate PR (or at least commit) please.

Reply:
dropped, can be improved in separate PR